src/main/java/com/aws/greengrass/clientdevices/auth/CertificateManager.java

```diff
@@ -298,15 +305,22 @@ public void configureCustomCA(CDAConfiguration configuration) throws InvalidConf
      * Uploads the stored certificates to the cloud.
      *
      * @param thingName Core device name
+     * @param certificates a list of X509Certificate to be uploaded to IoTCore
+     *
      * @throws CertificateEncodingException   If unable to get certificate encoding
      * @throws KeyStoreException              If unable to retrieve the certificate
      * @throws IOException                    If unable to read certificate
      * @throws DeviceConfigurationException   If unable to retrieve Greengrass V2 Data client
      */
     @SuppressWarnings({"PMD.AvoidCatchingGenericException", "PMD.AvoidRethrowingException"})
-    public void uploadCoreDeviceCAs(String thingName) throws CertificateEncodingException, KeyStoreException,
+    public void uploadCoreDeviceCAs(String thingName, List<X509Certificate> certificates)
```

Comment:
The cloud only supports a single cert, right? Can we change this method signature to reflect that?

Reply:
I think we could upload multiple but for CDA we should not need to upload more than one so, I'll patch it